discretisedfield/tests/test_field.py

```diff
@@ -179,19 +179,20 @@ class WrongType:
     def test_set_with_ndarray(self):
         for mesh in self.meshes:
             f = df.Field(mesh, nvdim=3)
-            f.value = np.ones(
-                (
-                    *f.mesh.n,
-                    f.nvdim,
+            f.update_field_values(
+                np.ones(
+                    (
+                        *f.mesh.n,
+                        f.nvdim,
+                    )
                 )
             )
```

Comment:
```suggestion
            f.update_field_values(np.ones((*f.mesh.n, f.nvdim)))
```

That should be short enough to fit in one line. No idea why black did not change this.